File: Python/1074 - Par ou Ímpar.py
N = int(input())

for _ in range(N):
    x = int(input())
    
    if x == 0:
        print("NULL")
    else:
        tipo = "EVEN" if x % 2 == 0 else "ODD"
        sinal = "POSITIVE" if x > 0 else "NEGATIVE"
        print(f"{tipo} {sinal}")

File: Clojure/1074 - Par ou Ímpar.clj
(let [N (Integer/parseInt (read-line))]
  (dotimes [_ N]
    (let [x (Integer/parseInt (read-line))]
      (if (= x 0)
        (println "NULL")
        (let [tipo (if (zero? (mod x 2)) "EVEN" "ODD")
              sinal (if (> x 0) "POSITIVE" "NEGATIVE")]
          (println (str tipo " " sinal)))))))